lang/en/texts/index-pro.html

```diff
@@ -35,6 +35,19 @@ <h2>Find opportunities to improve the quality of your products</h2>
 	<li>Significantly improve the Nutri-Score with a reasonable reduction in sugar, salt or saturated fatty acids.</li>
 	<li>Detect products with a lower nutritional profile than similar products on the market.</li>
 </ul>
+  
+ <h2><em>New!</em> &nbsp; Complete the data necessary to calculate the Eco-score for products sold in France.</h2>
```

Comment:
That sounds very weird. They won't see the Eco-Score unless they access fr.pro.openfoodfacts.org

I think it's best not to talk about the Eco-Score except on the French platform.

Reply:
It was a request from Hangy. I will delete mentions to the Eco-Score